### PR TITLE
Found and fixed some typing errors

### PR DESCRIPTION
**Pull request by soummo19**
#435
signed-off by: Soumyadeep Laskar (soumyadeeplaskar.2000@gmail.com)
Found some error in typing

### CODE REVIEW

**Comment by soummo19** [comment]
Please review my pr

File: CHAOSScon/2019EU/speakers.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ Ildikó started her journey with virtualization during the university years and
 
 Session: **Keynote 2: Finding the Order in the Chaos(s) of Metrics: Are We There Yet?**
 
-> We often ask ourselves and look around in our environments, wondering if our actions and activities are up to standards and efficient enough or if there is room for improvement. Measuring efficiency, modeling health with numbers, and creating dashboards have been part of our lives for a long time now, but do we do it in the right way? Are we abusing the numbers or are we looking at accurate pictures at all times?
+> We often ask ourselves and look around in our environments, wondering if our actions and activities are up to standards and efficient enough or if there is room for improvement. Measuring efficiency, modelling health with numbers, and creating dashboards have been part of our lives for a long time now, but do we do it in the right way? Are we abusing the numbers or are we looking at accurate pictures at all times?
```
Comment (germonprez) [inline CHAOSScon/2019EU/speakers.md:114]: Modeling is the right term here :) Could you please update?

File: About/About/faq.md [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
 Our goal is to define metrics independent of a specific implementation. At the CHAOSS project, we provide the tools by which you can make more informed decisions regarding the open source projects that are important to you -- in your specific context. 
 
-## What are the IP or commitments that contributing members are making to make sure that the derived IP is patent free and distributed under an open source license.
-Our goal is to make our innovations available to everyone to use for free. We honor open source licenses of upstream projects. We license our own work under the GPLv2+ and MIT open source licenses. We only accept patent free contributions or where patent licenses are available for free.
+## What are the IPs or commitments that contributing members are making to make sure that the derived IP is patent free and distributed under an open source license?
+Our goal is to make our innovations available to everyone to use for free. We honour open source licenses of upstream projects. We license our own work under the GPLv2+ and MIT open source licenses. We only accept patent free contributions or where patent licenses are available for free.
```
Comment (germonprez) [inline About/About/faq.md:14]: US vs. British terminology: honor vs. honour. Your suggestion is fine.

Comment (soummo19) [reply About/About/faq.md:14]: ok

File: CHAOSScon/2019EU/speakers.md (hunk 1 above)
Comment (soummo19) [reply CHAOSScon/2019EU/speakers.md:114]: ok

**Comment by soummo19** [comment]
Please review...

**Comment by germonprez** [comment]
Thanks for fixing the typos.

**Pull request merged**